sdk/javascript/src/SharedKey.js

```diff
@@ -0,0 +1,178 @@
+const { SharedKey256 } = require('./CryptoTypes');
+const { hkdf } = require('@noble/hashes/hkdf');
+const { sha256 } = require('@noble/hashes/sha256');
+const tweetnacl = require('tweetnacl');
+
+// order matches order of exported methods
+const {
+	crypto_verify_32, gf, pack25519, unpack25519, pow2523, set25519
+} = tweetnacl.lowlevel;
+
+// region curve operations - unfortunatelly tweetnacl.lowlevel does not expose those functions, so needed to copy them here
```

Comment:
ok to copy here, but why not export from tweetnacl version used by NEM ?

Reply:
i mean update that one to export more in lowlevel

Reply:
as mentioned on discord, imo it would look _weird_ if symbol version would use >two< tweetnacl imports.